### PR TITLE
Update irc file and images

### LINKED ISSUE #869: Update the IRC researchers list
**Is your feature request related to a problem? Please describe.**
We need to add the new IRC researchers' profiles to the IRC page

**Describe the solution you'd like**
Add the images taken from linkedin profiles.
Update assets/content/json/irc.json file

New profiles:
Name | Position | Field | Linkedin
| -- | -- | -- | -- |
G.T Upeka Vianthi Somaratne | Ph.D Candidate | Information Technology | https://www.linkedin.com/in/upekasomaratne-83749b4a/
Kajeenthan Kamalaseelan | Ph.D Candidate | Civil Engineering | linkedin.com/in/eng-kajeenthan-kamalaseelan
Rasika Eranda Ashoka Dissanayake | Ph.D Candidate | Chemical Science | https://www.linkedin.com/in/rasika-ea-dissanayake/
Lasara Kariyawasam | Ph.D Candidate | Psychology | https://www.linkedin.com/in/lasara-kariyawasam-154373145/
Durga Dharmadana | Ph.D Candidate | Biomedical Science | https://www.linkedin.com/in/durga-dharmadana-a8aa6687/
Upekha Pathumi Miriyagalla | Ph.D Candidate | Social Psychology | https://www.linkedin.com/in/upekha-miriyagalla-54754a93/
Biyanka Jayangani Ekanayake | Ph.D Candidate | Construction Automation | linkedin.com/in/biyanka-ekanayake-0146a8142
Pevindu Abeysinghe | Ph.D Candidate | Reproductive Biology | https://www.linkedin.com/in/pevindu-abeysinghe-a10085138/
Udani Abhisheka Wijewardhana | Ph.D Candidate | Statistics | https://www.linkedin.com/in/udani-wijewardhana-2134426b/
Sumudu Mallawaarachchi | Ph.D Candidate | Psychology | https://lk.linkedin.com/in/sumudu-mallawaarachchi-10a921133

**Describe alternatives you've considered**
N/A

**Additional context**
You can refer this thread. https://sef.discourse.group/t/updating-irc-page/171

### PR DESCRIPTION
**Pull request by nejawittatchy**
## Purpose

Update IRC files and Images
The purpose of this PR is to fix #869 

### Preview Link

https://pr-879-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
- Mac OS, Safari

### CODE REVIEW

**Review by Gimhan-minion — CHANGES_REQUESTED** [review]
@nejawittatchy Good Job! I guess you missed the LinkedIn links. Can you update them too as in the issue #868?

File: assets/content/json/irc.json [hunk 1]
```diff
@@ -1,4 +1,84 @@
...
+    "description": "Biomedical Science",
+    "image": "durga_dharmadana.jpeg",
+    "social": {
+      "linkedin": "akbandara"
+    }
+  },
+  {
+    "name": "Upeka Miriyagalla",
+    "description": "Social Psychology",
+    "image": "arosha.jpeg",
```
Comment (anjula-sack) [inline assets/content/json/irc.json]: name and the name of the image is different

File: assets/content/json/irc.json [hunk 2]
```diff
@@ -591,4 +671,5 @@
       "linkedin": "jayaruwangunathilake"
     }
   }
+
```
Comment (anjula-sack) [inline assets/content/json/irc.json:674]: Remove this line

File: assets/content/json/irc.json [hunk 3]
```diff
@@ -1,4 +1,84 @@
...
+    "name": "Rasika Dissanayake",
+    "description": "Chemical Science",
+    "image": "rasika_dissanayake.jpeg",
+    "social": {
+      "linkedin": "akbandara"
+    }
+  },
+  {
+    "name": "Lasara Kariyawasam",
+    "description": "Psycohlogy",
```
Comment (anjula-sack) [inline assets/content/json/irc.json]: Check the spellings

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by Gimhan-minion — APPROVED** [review]

**Review by anjula-sack — APPROVED** [review]
Awesome! @nejawittatchy

**Pull request merged**